### ISSUE TITLE
Piping output of jq to tr to get rid of quotes

### ISSUE THREAD

**Issue by JCGoran** [koalaman/shellcheck#2177]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

The `jq` program is sometimes used like this:

```sh
$ printf '"string\\\""' | jq '.' | tr -d '"'
string\
```

but the user probably intended the output to be something like this instead (and this seems to be a [frequent question](https://github.com/stedolan/jq/issues/1735)):

```sh
$ printf '"string\\\""' | jq -r '.'
string"
```

#### Here's what shellcheck currently says:

Nothing.

#### Here's what I wanted or expected to see:

Emit a warning to not use `tr`, but instead use the `-r` (`--raw-output`) flag of `jq`.
